node/pkg/logscribe/app.go

```diff
@@ -260,15 +262,16 @@ func (a *App) createGithubIssue(ctx context.Context, processedLogs map[string][]
 				log.Error().Err(err).Msg("Failed to create github issue")
 			}
 
-			err = db.QueryWithoutResult(ctx, "INSERT INTO processed_logs (log_hash) VALUES (@hash)", map[string]any{
-				"hash": hash,
-			})
-			if err != nil {
-				log.Error().Err(err).Msgf("Failed to insert processed log: %v", entry)
-			}
+			processedLogHashes = append(processedLogHashes, []interface{}{hash})
 
 			issueCount++
 		}
 	}
+
+	_, err := db.BulkCopy(ctx, "processed_logs", []string{"log_hash"}, processedLogHashes)
```

Comment:
please don't use bulk copy but there should be bulk insert or something else I wrote before with different name. bulk copy can be slower if there aren't much enough entries

Reply:
https://github.com/Bisonai/orakl/blob/951b6283e44d1531bcac36ef2440f8872db89b92/node/pkg/db/pgsql.go#L201